examples/showcase/.storybook/main.js

```diff
@@ -1,18 +1,53 @@
 /** @type { import('storybook-vue').StorybookConfig } */
 const config = {
+  framework: {
+    name: '@storybook-vue/nuxt',
+    options: {},
+  },
   stories: [
     '../components/**/*.mdx',
     '../components/**/*.stories.@(js|jsx|ts|tsx|mdx)',
     '../pages/**/*.stories.@(js|jsx|ts|tsx|mdx)',
   ],
+  core: {
+    builder: '@storybook/builder-vite',
+  },
   addons: [
-    '@storybook/addon-docs',
-    '@storybook/addon-links',
+    {
+      name: '@storybook/addon-docs',
+      options: {
+        csfPluginOptions: null,
+        mdxPluginOptions: {},
+      },
+    },
     '@chromatic-com/storybook',
+    '@storybook/addon-links',
   ],
-  framework: {
-    name: '@storybook-vue/nuxt',
-    options: {},
+  async viteFinal(config) {
```

Comment:
If these steps are really necessary, then they should be handled by the plugin. The user shouldn't have the need to overwrite the viteFinal method.